lib/runtime-c-api/src/lib.rs

```diff
@@ -451,21 +449,21 @@ pub unsafe extern "C" fn wasmer_module_instantiate(
 
         let namespace = namespaces
             .entry(module_name)
-            .or_insert_with(|| Namespace::new());
+            .or_insert_with(Namespace::new);
 
         let export = match import.tag {
             wasmer_import_export_kind::WASM_MEMORY => import.value.memory as *mut Export,
             wasmer_import_export_kind::WASM_FUNCTION => import.value.func as *mut Export,
             wasmer_import_export_kind::WASM_GLOBAL => import.value.global as *mut Export,
             wasmer_import_export_kind::WASM_TABLE => import.value.table as *mut Export,
         };
-        namespace.insert(import_name, unsafe { (&*export).clone() });
+        namespace.insert(import_name, (&*export).clone());
     }
     for (module_name, namespace) in namespaces.into_iter() {
         import_object.register(module_name, namespace);
     }
 
-    let module = unsafe { &*(module as *mut Module) };
+    let module = &*(module as *const Module);
```

Comment:
I _think_ it's not necessary to get a mutable module here. Correct me if I misunderstood the code!